draft-ietf-gnap-core-protocol.md

```diff
@@ -3261,12 +3215,106 @@ the token management URL as valid, since the end result is still the token
 not being usable.
 
 
-# Using Access Tokens {#use-access-token}
+# Securing Requests from the Client Instance {#secure-requests}
+
+In GNAP, the client instance secures its requests to the AS and RS by presenting an access
+token, presenting proof of a key that it possesses, or both an access token and
+key proof together.
```

Comment:
both an access token and key proof together.
-> key proof of possession ?

Reply:
This is "key proof" below as well, all of which mean "proof of possession of a specific key". Should we add a defined term for this, perhaps? At the moment it's at least consistent if we later replace with a better term.

Reply:
ok. If we see it is useful to add a definition, we can do that later on (right now I'd keep the current scope, since there are already quite a few changes, but I put that in my notes)